transform/binary-plugin/examples/test/whiteout-all/whiteout-all.go

```diff
@@ -1,21 +1,14 @@
 package main
 
 import (
-	"fmt"
-
 	jsonpatch "github.com/evanphx/json-patch"
 	"github.com/konveyor/crane-lib/transform"
 	"github.com/konveyor/crane-lib/transform/cli"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
 func main() {
-	u, err := cli.Unstructured(cli.ObjectReaderOrDie())
-	if err != nil {
-		cli.WriterErrorAndExit(fmt.Errorf("error getting unstructured object: %#v", err))
-	}
-
-	cli.RunAndExit(cli.NewCustomPlugin("WhiteoutAllPlugin", Run), u)
+	cli.RunAndExit(cli.NewCustomPlugin("WhiteoutPlugin", "v1", nil, Run))
```

Comment:
Possible bad merge with my plugin name fix. Should preserve the name "WhiteoutAllPlugin"